mergify_engine/actions/queue.py

```diff
@@ -57,6 +58,37 @@ def get_config_schema(
         cls,
         partial_validation: bool,
     ) -> typing.Dict[typing.Any, typing.Any]:
+        if config.ALLOW_COMMIT_MESSAGE_MODE:
+            return {
+                voluptuous.Required(
+                    "name", default="" if partial_validation else voluptuous.UNDEFINED
+                ): str,
+                voluptuous.Required("method", default="merge"): voluptuous.Any(
+                    "rebase", "merge", "squash"
+                ),
+                voluptuous.Required("rebase_fallback", default="merge"): voluptuous.Any(
+                    "merge", "squash", "none", None
+                ),
+                voluptuous.Required(
+                    "merge_bot_account", default=None
+                ): types.Jinja2WithNone,
+                voluptuous.Required(
+                    "update_bot_account", default=None
+                ): types.Jinja2WithNone,
+                voluptuous.Required("update_method", default="merge"): voluptuous.Any(
+                    "rebase", "merge"
+                ),
+                voluptuous.Required("commit_message", default="default"): voluptuous.Any(
+                    "default", "title+body"
+                ),
+                voluptuous.Required(
+                    "commit_message_template", default=None
+                ): types.Jinja2WithNone,
+                voluptuous.Required(
+                    "priority", default=queue.PriorityAliases.medium.value
+                ): queue.PrioritySchema,
+                voluptuous.Required("require_branch_protection", default=True): bool,
+            }
```

Comment:
I think i can optimize this part to avoid having duplicate voluptuous, and create validator object as the same as merge file.

Reply:
You should! 😉